### ISSUE TITLE
plasma-applet-weather-widget 1.6.9 DEB packages

### ISSUE THREAD

**Issue by varlesh** [kotelnik/plasma-applet-weather-widget#53]
Maybe it's help for some users :)
Compiled for Xenial (Ubuntu 16.04)
amd64:
https://launchpad.net/~varlesh-l/+archive/ubuntu/ubuntu-tools/+files/plasma-applet-weather-widget_1.6.9_amd64.deb
i386:
https://launchpad.net/~varlesh-l/+archive/ubuntu/ubuntu-tools/+files/plasma-applet-weather-widget_1.6.9_i386.deb

**Comment by kotelnik** [comment]
Nice, thanks! :)

**Comment by shmerl** [comment]
Is anyone planning to package it for Debian?

**Comment by varlesh** [comment]
Updated to 1.6.10 version for Xenial

**Comment by gour** [comment]
The package does not work for me on Debian Sid. :-( Anyone aware of some other source?

And, yes, default weather report does not list my city...on GNOME I was using openweathermap shell-extension which works nicely...

**Comment by varlesh** [comment]
Try package for bionic:
https://launchpad.net/~varlesh-l/+archive/ubuntu/bionic/+packages

**Comment by gour** [comment]
@varlesh nope, it does not work. I've even tried to build it from the source and it goes OK, but then I hit #71. Any hint in regard?

**Comment by varlesh** [comment]
@gour For build i'm install this packages:
`g++ plasma-framework-dev plasma-workspace-dev libkf5declarative-dev libkf5globalaccel-dev libkf5configwidgets-dev libkf5xmlgui-dev libkf5windowsystem-dev kdoctools-dev cmake extra-cmake-modules kdelibs5-dev qt5-qmake qt5-default`

**Comment by gour** [comment]
@varlesh Thank you. That helped!! I've installed those which were missing and they pulled bunch of other packages. Strange thing is that the build was not failing before, but the widget was obviously missing some runtime deps.

In any case, it makes me enthusiastic to provide proper package for Debian (Sid).  ;)
